flixel/system/render/hardware/gl/GLRenderHelper.hx

```diff
@@ -0,0 +1,392 @@
+package flixel.system.render.hardware.gl;
```

Comment:
do you think it's better to have 3 packages: `blit`, `tile` and `gl` on the same level?
Maybe it's good idea. Thanks!